tools/scripts/upload-to-nr.js

```diff
@@ -5,8 +5,6 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-var fs = require('fs')
-var path = require('path')
 var request = require('request')
```

Comment:
I am going to open a backlog tech debt ticket to remove our reliance on this third-party. The package has been deprecated for some time: see [here](https://github.com/request/request/issues/3142). Node provides a supported package, [Undici](https://github.com/nodejs/undici), that we should use instead.

Reply:
https://issues.newrelic.com/browse/NR-73098

Reply:
could also switch to using native fetch which is included in Node if we bump the node versions inside the GHA containers